src/sentry/static/sentry/less/icon-arrow.less

```diff
@@ -0,0 +1,11 @@
+/**
+ * Icons
+ * ============================================================================
+ */
+
+.icon-arrow-right,
+.icon-arrow-left {
+  font-size: 16px !important;
+  position: relative;
+  top: 2px;
+}
```

Comment:
Don’t want to go on a quest to find out where this is used and delete it. Ideally these styles should be in Emotion.

Reply:
we still have those in templates :/